game_board/ai/ai_handler.py

```diff
@@ -0,0 +1,172 @@
+""" AI for CMSC 447 DSG, Fall 2020
```

Comment:
Note: next sprint we want heuristic for awarding golden node to be closer to the root.